README.md

```diff
@@ -351,3 +351,25 @@ You can create your on commands by using the lua functions this plugin provides
 command! -count=1 TermGitPush  lua require'toggleterm'.exec("git push",    <count>, 12)
 command! -count=1 TermGitPushF lua require'toggleterm'.exec("git push -f", <count>, 12)
 ```
+
+### Open multiple terminals side-by-side
+| Direction | Tested |
+| --- | --- |
+| vertical | ✔️ |
+| horizontal | ✔️ |
+| window | ⚠️ |
+| float | ⚠️ |
+
+✔️: tested, working.
+
+⚠️: not tested, may not working.
+
+On your first terminal, you need to escape the `TERMINAL` mode via the `<Esc>` key that has been mapped with the combination `<C-\><C-n>`.
```

Comment:
This is a minor nit but you can improve the rendering of a keyboard character using `<kbd>Esc</kbd>` which gives <kbd>Esc</kbd>, or `<kbd>C-\\</kbd><kbd>C-N</kbd>` which gives <kbd>C-\\</kbd><kbd>C-N</kbd>